seed/views/v3/columns.py

```diff
@@ -111,6 +113,46 @@ def list(self, request):
             'columns': columns,
         })
 
+    @api_endpoint_class
+    @ajax_request_class
+    def create(self, request):
+        organization_id = self.get_organization(self.request)
+
+        table_name = self.request.data.get("table_name")
+        if table_name != "PropertyState" and table_name != "TaxLotState":
+            return JsonResponse({
+                'status': 'error',
+                'message': 'table_name must be "PropertyState" or "TaxLotState"'
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        try:
+            new_column = Column.objects.create(
+                is_extra_data=True,
+                **self.request.data
+            )
+            new_column.save()
+        except ValidationError as e:
+            return JsonResponse({
+                'status': 'error',
+                'message': str(e)
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        state_model = PropertyState if table_name == 'PropertyState' else TaxLotState
+        states = state_model.objects.filter(organization_id=organization_id)
+        states.update(
+            extra_data=Func(
+                F("extra_data"),
+                Value([self.request.data['column_name']]),
+                Value(1, JSONField()),
```

Comment:
I noticed that when I create a new column it starts out with a value of 1 for every property across cycles. It seems like that is coming from this line. I would expect the new column to be created without any data in it otherwise you have to go back and edit all of the properties to remove the value of 1.

Additionally, it might be useful to give the user the ability to auto-populate the new column with a certain value specified by the user, but that seems like it would be a new ticket if that's even something that users want. @RDmitchell  can you weigh in on this?